check_stability.py

```diff
@@ -427,7 +427,9 @@ def write_inconsistent(inconsistent, iterations):
 def write_results(results, iterations):
     logger.info("## All results ##\n")
     for test, test_results in results.iteritems():
-        logger.info("### %s ###" % test)
+        baseurl = "https://w3c-test.org/submissions"
```

Comment:
You need to test for `.https.` in the filename to decide if it's a http or https url.